bin/setup

```diff
@@ -19,6 +19,20 @@ echo
 
 bundle install --binstubs
 
+mh_ui_testing_release=0.9.4
```

Comment:
I can't decide if this is a good idea or not since, at the moment, there's still going to be a good bit of churn in that project.